components/homepage/Hero.tsx

```diff
@@ -22,24 +22,15 @@ const Hero = (props: Props) => {
           to excellence
         </h4>
         <div className="flex flex-col md:flex-row md:gap-[1.13rem]">
-          <button className="herobtn md:herobtnlg mb-[.88rem] flex rounded-[379.77006rem] bg-blue dark:bg-blue1 md:mb-0">
-            <p className="herotext text-white md:text-[1.125rem]">My Work</p>
-          </button>
-          <div className="herocopy md:herocopylg mb-[1.69rem] flex bg-white dark:bg-darkbg2 md:mb-0 md:w-fit">
-            <p className="herocopytext text-slate dark:text-white1">
-              austin@fcvt.dev
-            </p>
-            <Image
-              className="shrink-0"
-              src="/Frame.svg"
-              width={16}
-              height={16}
-              alt="Copy"
-            />
-          </div>
+          <Link href="/portfolio">
+            <button className="herobtn md:herobtnlg mb-[.88rem] flex rounded-[379.77006rem] bg-blue dark:bg-blue1 md:mb-0">
+              <p className="herotext text-white md:text-[1.125rem]">My Work</p>
+            </button>
+          </Link>
+          <Email />
         </div>
       </div>
-      <div className="heroimg md:heroimglg md:flex-row">
+      <div className="heroimg md:heroimglg md:w-1/2 md:flex-row">
         <Image src="/Hero.svg" width={757} height={541} alt="Hero computers" />
```

Comment:
```suggestion
        <Image src="/Hero.svg" width={757} height={541} alt="Hero computers" priority/>
```
This hero image is the main image you want to load as the website renders. So label it as priority to optimize page load.

https://nextjs.org/docs/pages/building-your-application/optimizing/images

![image](https://github.com/austinmargarone/portfolio/assets/4108484/34926c3f-e3a5-4d7d-bd0b-f2303fbca60e)